x-pack/plugins/saved_objects_tagging/public/components/edition_modal/create_or_edit_modal.tsx

```diff
@@ -184,6 +192,7 @@ export const CreateOrEditModal: FC<CreateOrEditModalProps> = ({
               resize="none"
               fullWidth={true}
               compressed={true}
+              aria-describedby={optionalMessageId}
```

Comment:
The `labelAppend` setup in EUI is only a visual treatment because EUI can't make any assumptions about what might be put in there. Because we're adding instructions about the `textarea` we need to hook those up.

Reply:
So, in effect, we're hooking up two labels to this textarea: Description and Optional, right?

Reply:
More or less. There's a slight difference difference between `aria-label`/`aria-labelledby` and `aria-describedby` but the result is similar.

When actually navigating through the page, it gets read out one after the other: "{label} _slight pause or a tone_ {description}"

The biggest difference is when using something like VoiceOver's Rotor (different names in NVDA and JAWS) which can list all buttons on the page: it goes through them by label and the description isn't available until you select the button you want to jump to.

So, best practice, is to put auxiliary content into descriptions (such as instructions or examples) but to put names into the label. (`aria-labelledby` can take a list of ids so you can construct long names when appropriate because a long name with good context is usually better than a short name missing context.)